frontend/src/common/queries/README.md

```diff
@@ -60,9 +60,9 @@ The benefits of this arrangement include:
    code to achieve that:
 
    ```ts
-   import { useMutation, useQueryCache } from "react-query";
+   import { useMutation, useQueryClient } from "react-query";
 
-   const queryCache = useQueryCache();
+   const queryClient = useQueryClient();
```

Comment:
https://react-query.tanstack.com/guides/migrating-to-react-query-3#the-querycache-has-been-split-into-a-queryclient-and-lower-level-querycache-and-mutationcache-instances